.github/workflows/typecheck.yml

```
@@ -0,0 +1,45 @@
+jobs:
+  type-check:
+    name: Type Check (tsc)
+    runs-on: ubuntu-latest
+    steps:
+      - name: 📦 Checkout project repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 📦 Setup Node + Bun + Install Dependencies
+        uses: ./.github/actions/setup-node-bun-install
```

actionlint

description is required in metadata of "Setup Node + Bun + Install Dependencies" action at "/inmem/42/b29381e2-8ea3-43ee-a706-48cc81892d34/home/jailuser/git/.github/actions/setup-node-bun-install/action.yml"

(action)